python/test/test_publish.py

```diff
@@ -10,6 +11,7 @@
 from publish.unittestresults import get_test_results
 from test import d, n
 
+THIS_DIR = os.path.dirname(__file__)
```

Comment:
lets go for a variable for the path to the test files:

```suggestion
test_files_path = os.path.dirname(os.path.join(__file__, 'files'))
```

If that `test_files_path` is even a `pathlib.Path`, then those verbose `os.path.join(test_files_path, 'no-cases.xml')` beome `test_files_path / 'no-cases.xml'`. Though you may have to wrap it with a `str()` where the string path is needed.

Reply:
Yeah, I'm gonna go for a wrapping everywhere but that's doable